package.json

```diff
@@ -88,24 +90,36 @@
     "yargs": "^16.0.0"
   },
   "devDependencies": {
-    "@netlify/eslint-config-node": "^3.2.8",
+    "@ava/typescript": "^2.0.0",
+    "@netlify/eslint-config-node": "github:netlify/eslint-config-node#feat/typescript",
     "adm-zip": "0.5.5",
     "ava": "^3.0.0",
     "cpy": "^8.0.0",
+    "esm": "^3.2.25",
```

Comment:
Can you clarify why this is needed?

Reply:
https://github.com/netlify/zip-it-and-ship-it/pull/675#issuecomment-930057320

Reply:
This is needed, because `@ava/typescript` doesn't transpile the "import {} from ..." calls in our test files. So we need to teach Node.js what to do with them, and `esm` does that.

Reply:
esm and @ava/typescript Were removed in 1842943